website/docs/tutorials/06-deploying/02-migrations/declare.md

```diff
@@ -0,0 +1,28 @@
+# `declare`
+
+```python
+def declare(contract_path: str) -> DeclaredContract:
+
+class DeclaredContract:
+    class_hash: int
+```
+Declares contract given a relative to a project root path to **compiled** contract.
+
+:::info
+This cheatcode has testing counterpart — [declare](/docs/tutorials/testing/cheatcodes/declare).
```

Comment:
It has not right now, interfaces differ. I think we should not provide such information to avoid confusion. After we unify those interfaces we can add it again